examples/circuitplay/main.go

```diff
@@ -0,0 +1,90 @@
+// This example is intended to be used with the Adafruit Circuitplay Bluefruit board.
+// It allows you to control the color of the built-in NeoPixel LEDS while they animate
+// in a circular pattern.
+//
+package main
+
+import (
+	"image/color"
+	"machine"
+	"time"
+
+	"tinygo.org/x/bluetooth"
+	"tinygo.org/x/drivers/ws2812"
+)
+
+var adapter = bluetooth.DefaultAdapter
+
+// TODO: use atomics to access this value.
+var ledColor = [3]byte{0xff, 0x00, 0x00} // start out with red
+var leds [10]color.RGBA
+var hasColorChange = true
```

Comment:
`hasColorChange` isn't used, was that intentional?

Reply:
Removed.